airbyte-ci/ci/ci_credentials/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+[tool.poetry]
+name = "ci_credentials"
+version = "1.1.0"
+description = "CLI tooling to read and manage GSM secrets"
+authors = ["Airbyte <contact@airbyte.io>"]
+readme = "README.md"
+packages = [{ include = "ci_credentials" }]
+
+[tool.poetry.dependencies]
+python = "^3.10"
```

Comment:
Python 3.10 all around!!!

Reply:
ci_credentials is installed on GHA workflows that are running `3.9`. Can we keep compatibility of `ci_credentials` with `3.9` while suggesting use of `3.10` though `.python-version` files?

Reply:
I believe we can!

Reply:
Is there a reason to keep the version diff between local and ci? Could this be another source of behavior mismatches?

Reply:
nvm, seems like we bumped most things as we went along, except `.github/workflows/connector-performance-command.yml` for some reason?

Reply:
Oh I think that was a miss on my part.

I ended up going with 3.10 everywhere as it seems that there were no issues encountered while running these tools as 3.10